lib/diagnostic.rb

```diff
@@ -5,28 +5,34 @@
 #
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
+ruby lib/example.rb
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'pry'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
+pry
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'name = "star wars episode viii"'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+name = "star wars episode viii"
+puts name
+
+Diagnostic.variable =  "character = #{'Jar Jar Binks'}"
 
```

Comment:
you only want to interpolate one thing and also you seem to be assigning it to a variable. this will blow up.

right answer : Diagnostic.interpolation = "#{character} is really a Sith Lord."